spec/order_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'order'
+
+describe Order do
+  let(:item) { double:item }
+  let(:quantity) { double:quantity }
```

Comment:
Lint/SpaceBeforeFirstArg: Put space between the method name and the first argument.